helpers/kes.css

```diff
@@ -1,321 +1,404 @@
+.kes-update-available {
+    color: var(--kbin-alert-info-text-color);
+    position: relative;
+    right: 1.5rem;
+    bottom: 0.3rem;
+}
+
+span.kes-update {
+    display: inline-block;
+    position: absolute;
+}
+
+#kes-settings:hover .kes-update-available {
+    color: var(--kbin-success-color);
+}
+
 #kes-settings-button {
-     top: 0;
-     right: 0;
-     padding: 3px 1rem 0;
-     min-width: 3rem;
-     color: var(--kbin-header-link-color);
-     cursor: pointer;
-}
- #kes-settings {
-     border-bottom: 3px solid transparent;
-}
- #kes-settings:hover {
-     border-bottom: var(--kbin-header-hover-border);
-     cursor: pointer;
-}
- #kes-settings-button:hover {
-     color: var(--kbin-header-link-hover-color);
-}
- .kes-transparent-mode-modal {
```

Comment:
Looks like this part got dropped

Reply:
Hmm, I didn't see that in my files. Was it added today?

Reply:
Yes, probably an hour or two ago now. Not sure how this got past. We can also enable a setting in the repository to force PRs to include the latest commits or they get rejected, although I *think* this needs a CI pipeline on a server to work properly, because it expects some kind of git status callback. I haven't played with the settings on it that much.

Reply:
I think we'd notice a huge decrease in issues if linting was standardized. Wouldn't show PRs with huge style changes so things wouldn't be missed as much.

Reply:
100% correct. Just went in and fixed it. Was carrying around this linting debt but kicking the can down the road.

Reply:
vimrc:
```
let g:ale_fixers = {'rust': ['rustfmt', 'trim_whitespace', 'remove_trailing_lines'],'c': ['clang-format'], 'javascript': ['eslint']}
let g:ale_fix_on_save = 1
```

.eslintrc
```
{
  "root": true,
  "parserOptions": {
    "ecmaVersion": 2021,
      "sourceType": "module"
  },
  "extends": "eslint:recommended",
  "rules": {
      "space-before-function-paren": ["error", "always"],
      "space-before-blocks": ["error", "always"]
  }
}
```

Added these rules per the syntax I saw coming from your code:
https://eslint.org/docs/latest/rules/space-before-blocks
https://eslint.org/docs/latest/rules/space-before-function-paren

Any others you wish to add?
Any ECMA version preference?

Reply:
I didn't see this reply - apologies. Perhaps this will help, obviously can be changed over time: https://github.com/aclist/kbin-kes/pull/122

Reply:
No worries, it seems like inline conversations are the first thing to get buried for some reason. Do you get email notifications for them?

Reply:
I do, but GitHub is rather spammy lol.

Reply:
Mail filters!